### ISSUE TITLE
Feature Request: customize shutter and power buttons  

### ISSUE THREAD

**Issue by salarmehr** [gopro/labs#205]
Could you please add a feature to change the function of the power button and shutter buttons? My main need is to be able to use the power button as a shutter button and also disable the mod change feature of it so I can turn on and off the camera with a quick single press. Here are some options that can be defined for each key

- power 
- shutter (image)
- start/stop video capture
- change mod
- switch video mode (hyper view, superview, ...) with each press (on image view switched the image mode)
- switch delay timder 
- ....

Ideally, you can define a function for double-pressing a button and long-pressing a button. 
The length of the long press can be adjustable too.

**Comment by salarmehr** [comment]
These can be incremental. First, a simple option to disable mod change on the power press and make it an instant power switch. Next an option to switch the shutter and power functions.

**Comment by dnewman-gpsw** [comment]
yes, archive mode it available.  Also boot command could help. Ask in discussions.
